test_crates/trait_associated_type_now_doc_hidden/old/src/lib.rs

```diff
@@ -0,0 +1,33 @@
+// Basic test case
+pub trait PUB_TRAIT_A {
+    type TYPE_A;
+    // Should not flag already #[doc(hidden)] type
+    #[doc(hidden)]
+    type DOC_HIDDEN_TYPE;
+}
```

Comment:
nit: Could you rename the traits and associated types to the idomatic `CapitalizedSnakeCase` style instead of using `SCREAMING_SNAKE_CASE`? Only constants in Rust use that, so it's a bit disorienting to see it elsewhere.